designsafe/static/scripts/projects/components/manage-project/manage-project.component.js

```diff
@@ -284,6 +305,24 @@ class ManageProjectCtrl {
         let options = optionsList.filter(type => type != 'Other')
         return !options.includes(input) && typeof input !== 'undefined'
     }
+
+    nhRequired() {
+        if (this.project.value.projectType === 'field_recon'){
+            return true
+        } 
+        // else if (this.form.nhEvent.$dirty)
+        //     return [console.log('nhEvent:' + true), true]
+        // else if (this.form.$watch('nh-event', function(newVal, oldVal) {
+        //     if(newVal !== '') {
+        //         return [console.log('watch:' + true), true]
+        //     }
+        // })
+        // })
+        // else {
+        //     return false
+        // }
+        
+    }
```

Comment:
I don't think we need this function anymore

Reply:
removed the whole function